src/vivarium_dcpn_hypertension_mgt/tools/utilities.py

```diff
@@ -48,7 +48,7 @@ def patch_external_data(artifact_path: Path):
 
     for k, data_file in zip(external_data_keys, external_data_files):
         data = prep_external_data(data_file, location)
-        if data.empty:
+        if data.empty and data.index.empty:
```

Comment:
so, prep_external can now give you a df with an index and no data? is that not worth of a warning?

Reply:
I don't think so because it's the same behavior we have in vivarium_inputs where things like age_bins will end up as just an index because there are no value columns